components/builder-api/src/server/helpers.rs

```diff
@@ -69,6 +76,19 @@ pub fn package_results_json<T: Serialize>(packages: &[T],
     serde_json::to_string(&results).unwrap()
 }
 
+pub fn channel_listing_results_json<T: Serialize>(channel: &str,
+                                                  target: &str,
+                                                  packages: &[T])
+                                                  -> String {
+    let results = ChannelListingResults { channel: channel.to_string(),
+                                          target:  target.to_string(),
+                                          data:    packages, };
+    let r = serde_json::to_string(&results).unwrap();
+    trace!("LOGLOG-t {} {} {}", channel, target, r);
```

Comment:
Could we update the trace prefix message to be more descriptive about where this log is occuring?